test/BaseClass.cpp

```diff
@@ -0,0 +1,53 @@
+#define Adafruit_MAX31865_Test Adafruit_MAX31865_Base
+#include "Common.cpp"
+
+unittest(testingClassName) {
+  std::cout << std::endl;
+  Adafruit_MAX31865_Test thermo(ss, mosi, miso, sck);
+  assertEqual("Adafruit_MAX3186_Base", thermo.className());
+  std::cout << std::endl;
+}
+
+unittest(read_RTD) {
```

Comment:
Why are these tests in `BaseClass.cpp` instead of `Common.cpp`? We might have discussed this but I forgot so a comment explaining it would be helpful.